### PR TITLE
Replaced the source code with Doom 1993

### PR DESCRIPTION
**Pull request by orkwitzel**
"But can it run doom?"

### CODE REVIEW

**Comment by TheDokT0r** [comment]
"for legal reasons that's a joke"

**Comment by BrunoMS13** [comment]
looks good to me

**Comment by SSolarite** [comment]
lgtm!!

**Review by MJHC — COMMENTED** [review]
lgtm

**Comment by Illapavan** [comment]
lgtm

**Comment by MfellnerDev** [comment]
Lgtm 👍🏻

**Comment by acquitelol** [comment]
lgtm

**Comment by jake0sand** [comment]
lgtm

**Comment by litten2up** [comment]
10/10 idea

**Comment by okaits** [comment]
lgtm

**Comment by oliexe** [comment]
lgtm

**Comment by PttsDev** [comment]
lgtm

**Comment by hailer24** [comment]
Lgtm

**Comment by GetPsyched** [comment]
LGTM

**Comment by gotlougit** [comment]
LGTM

**Comment by Cryptizism** [comment]
lgtm, push to prod

**Comment by GamehunterKaan** [comment]
Elon, if you're the meme lord you know what to do.

**Review by GamehunterKaan — COMMENTED** [review]
For the memes

**Review by gotlougit — COMMENTED** [review]
The code seems alright to me, think this will compile, although we should probably get a CI in place for this for future PRs

**Comment by MoritzHayden** [comment]
```text
 _     _____ ________  ___
| |   |  __ \_   _|  \/  |
| |   | |  \/ | | | .  . |
| |   | | __  | | | |\/| |
| |___| |_\ \ | | | |  | |
\_____/\____/ \_/ \_|  |_/                       
```

**Comment by p4kl0nc4t** [comment]
LGTM.

**Comment by Routable** [comment]
LGTM

**Comment by josephcarragher** [comment]
lgtm.

**Comment by EmberHext** [comment]
LGTM

**Comment by dreyfus92** [comment]
LGTM ✅

**Comment by TaeKwonZeus** [comment]
LGTM

**Comment by joaoburatto** [comment]
LGTM

**Comment by TaylorNAlbarnaz** [comment]
lgtm

**Comment by Mechar418** [comment]
lgtm

**Comment by vgArchives** [comment]
LGTM!

**Comment by cheetosysst** [comment]
lgtm

**Review by vinevax — COMMENTED** [review]
LGTM

**Comment by StormVanDerPol** [comment]
LGTM!

**Comment by kenan238** [comment]
LGTM! merge to master, ready for prod

**Comment by zTags** [comment]
lgtm

**Comment by x8c8r** [comment]
lgtm

**Comment by expected-ingot** [comment]
LGTM!

**Comment by kenan238** [comment]
31 comments agreeing to merge to main. waiting for the contributors to merge

**Comment by iChillieman** [comment]
Bro! XD

**Comment by shim1998** [comment]
lgtm

**Comment by kutilang28** [comment]
lgtm

**Comment by HFDan** [comment]
LGTM

**Comment by Ewie21** [comment]
lgtm

**Comment by jeffersfp** [comment]
let's get that mayonaise

**Comment by cosmiccoincidence** [comment]
This PR needs an 'Improvement' label.

**Comment by narutopig** [comment]
LGTM

**Comment by mutexdeveloper** [comment]
switch to freedoom!

**Comment by mikoxyz** [comment]
LGTM

**Comment by vodam46** [comment]
LGTM!

**Comment by echnobas** [comment]
lets gamble, try merge

**Comment by wayne1512** [comment]
ready to merge

**Review by ErtanTaner — COMMENTED** [review]
LGTM

**Comment by xandermann** [comment]
LGTM

**Comment by lordbagel42** [comment]
> No more algorithm, then.

This is the new algorithm my man. Lets go for it.

LGTM!

**Comment by Syntoxr** [comment]
LGTM!

**Review by GDAndres98 — COMMENTED** [review]
LGTM

**Comment by joaoburatto** [comment]
F

**Comment by Staninna** [comment]
F

**Comment by Stojak139808** [comment]
I guess it can't run doom after all.

**Review by NoelisTired — COMMENTED** [review]
i approve

**Comment by prashanth-up** [comment]
LGTM!

**Comment by ltzmax** [comment]
LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

LGTM

**Comment by Vizxe** [comment]
lgtm

**Comment by svki0001** [comment]
LGTM

**Comment by icanwalkonwater** [comment]
lgtm

**Comment by darkpixlz** [comment]
lgtm

**Comment by vstyler96** [comment]
LGTM!

**Comment by RomainClem** [comment]
Why is this closed? This PR literally fixes everything.

**Comment by wiichicken** [comment]
thanks for doing what most of us are all too cowardly to do

**Comment by ElusiveByte** [comment]
Works like a charm, would recommend, 10/10 IMDB.
 
I mean, ...

LGTM

**Comment by litten2up** [comment]
> Works like a charm, would recommend, 10/10 IMDB.
> 
> I mean, ...
> 
> LGTM

oops, lgtm

**Comment by davidsuarezcdo** [comment]
LGTM

**Review by Anderlyn — COMMENTED** [review]
LGTM

**Comment by okfineIGitIt** [comment]
lgtm

**Comment by SyafaHadyan** [comment]
LGTM

**Comment by paxperscientiam** [comment]
Cut and print.

**Comment by kristjannomm** [comment]
lgtm

**Comment by Rohan-Muslekar** [comment]
I approve!

**Comment by RoneloSolo** [comment]
lgtm

**Comment by fluentmoheshwar** [comment]
LGTM

**Comment by Youda008** [comment]
There is no Doom source code in this MR, please fix.

**Comment by lorranpego** [comment]
LGTM, should we reopen?

**Comment by lMartin3** [comment]
LGTM! Reopen the issue

**Comment by GabrielHoy** [comment]
LGTM, should reopen issue 👍

**Comment by shanejones** [comment]
Ship it.

**Comment by mutexdeveloper** [comment]
reopen issue!!

**Comment by JustYuuto** [comment]
lgtm

**Review by orhanerday — COMMENTED** [review]
LGTM

**Comment by saedyousef** [comment]
LGTM ✅

**Review by saedyousef — COMMENTED** [review]
Add more tests please 🤣

**Comment by NikarashiHatsu** [comment]
LGTM!

**Comment by Nightcaat** [comment]
> Why is this closed? This PR literally fixes everything.

The head repository was deleted by the author

**Review by Nirzak — COMMENTED** [review]
LGTM

**Comment by faytekin** [comment]
LGTM

**Comment by DoctaEnkoda** [comment]
Lgtm

**Comment by RaphaelBRodrigues** [comment]
lgtm

**Comment by retbleed** [comment]
LGTM

**Comment by kkoukakis** [comment]
Looks good! 
LGTM! 
🚀

**Comment by oachkatzalschwoaf** [comment]
LGTM!

**Comment by JozefBn** [comment]
lgtm

**Comment by hmes98318** [comment]
LGTM

**Comment by davidleitw** [comment]
lgtm

**Comment by leang97** [comment]
LGTM, great algorithm, optimized for every device.

**Comment by M4th3m4tic4l** [comment]
LGTM

**Comment by milesueee** [comment]
lgtm👍

**Comment by arifrgilang** [comment]
lgtm

**Comment by acidmanifesto** [comment]
Lgtm

**Comment by awcator** [comment]
Lgtm

**Comment by 1chooo** [comment]
LGTM

**Comment by Stephen-Kamau** [comment]
LGTM

**Comment by pontakornth** [comment]
LGTM

**Comment by JZGoopi** [comment]
LGTM

**Review by pyxfluff — COMMENTED** [review]
LGTM.

**Comment by trylovetom** [comment]
LGTM ^^y

**Comment by zhaospei** [comment]
LTGM

**Comment by PalmDevs** [comment]
Dear @TheDokT0r,

I am writing to you today to urge you to reconsider the decision to close the pull request in question. This pull request is critical to the success of the project, and I believe that it deserves a second look.

As you are aware, you have put a lot of time and effort into this pull request. The code you have written is well-structured, efficient, and thoroughly documented. Despite that, the pull request was closed for absolutely no reason.

In addition, it is worth noting that this pull request aligns with the project's goals and objectives. You have taken the time to understand the project's needs and has developed a solution that addresses them. Furthermore, reopening this pull request could provide an opportunity for more constructive feedback and collaboration.

In conclusion, I implore you to reconsider the decision to close this pull request. You have put a lot of time and effort into this pull request, and it aligns with the project's goals and objectives. By reopening it, we can enhance the success of the project.

##### - ChatGPT

**Comment by wasurocks** [comment]
lgtm

**Comment by Manopphysics** [comment]
lgtm

**Comment by geceba** [comment]
lgtm

**Comment by Rechyrex** [comment]
_The only thing they fear is you_

LGTM :rocket:

**Comment by AngeloAugustoCSW** [comment]
do it please

LGTM

**Comment by ZsoltSafrany** [comment]
LGTM

**Comment by Nick-vdM** [comment]
LGTM

**Comment by okaits** [comment]
128 lgtm comments. plz reopen and merge it.

**Comment by aldoeliacim** [comment]
LGTM !

**Comment by wearrrrr** [comment]
LGTM

**Comment by Freeplayg** [comment]
LGTM

**Comment by ProudlyTM** [comment]
LGTM

**Review by SyafaHadyan — COMMENTED** [review]
LGTM

**Comment by rezizedginidze** [comment]
LGTM

**Review by vstyler96 — COMMENTED** [review]
```
 _     _____ ________  ___
| |   |  __ \_   _|  \/  |
| |   | |  \/ | | | .  . |
| |   | | __  | | | |\/| |
| |___| |_\ \ | | | |  | |
\_____/\____/ \_/ \_|  |_/                       
```

File: Doom1.WAD
Comment (SyafaHadyan) [inline Doom1.WAD:1]: looks good

**Review by SyafaHadyan — COMMENTED** [review]
LGTM

**Comment by UwUtisum** [comment]
now this is perfect

**Comment by darkpixlz** [comment]
Ready for production.

**Comment by jessiejs** [comment]
LGTM

**Review by okaits — COMMENTED** [review]
nice.

**Review by 1chooo — COMMENTED** [review]
LGTM🫡

**Review by 1chooo — COMMENTED** [review]
LGTM🫡

**Comment by jeromepalayoor** [comment]
LGTM

**Comment by reniivali** [comment]
LGTM!

**Comment by mm12** [comment]
lgtm

**Comment by azaleacolburn** [comment]
Much more useful and beneficial to society now, LGTM

**Comment by egeres** [comment]
LGTM

**Comment by telnettrauma** [comment]
the party ended an hour ago and hes still here

**Comment by gitnickolson** [comment]
LGTM

**Comment by bengangllipson** [comment]
LGTM

**Comment by the-bokya** [comment]
We need this in https://github.com/frappe/frappe
cc: @iamejaaz

**Review by alboxer2000 — COMMENTED** [review]
lgtm

**Comment by Mishatopkek** [comment]
lgtm
